src/formula/string_utils.cpp

```diff
@@ -356,3 +356,43 @@ std::string vngettext_impl(const char* domain,
 	const std::string msg = utils::interpolate_variables_into_string(orig, &symbols);
 	return msg;
 }
+
+int approximate_string_distance(const std::string &str_1, const std::string &str_2)
+{
+    int approximate_distance = 0;
+    if(str_1.length() == 0) {
+        return str_2.length();
+    }
+    else if(str_2.length() == 0) {
+        return str_1.length();
+    }
+    else {
+        int Lmax = 0;
```

Comment:
Local variables (or probably any variables) shouldn't start with an uppercase letter. I suggest `l_max` instead, or perhaps `len_max` since lowercase L might be confusing.

Reply:
Now that I gave it a thought, Lmax is only used once, so I will axe it and do `i < std::max(str_1.length(),str_2.length())` directly in the loop condition.